src/utils/__tests__/getPriceGuidance.test.ts

```diff
@@ -14,42 +14,46 @@ describe("#getPriceGuidance", () => {
     const results = {
       marketingCollection: {
         artworks: {
-          hits: [
-            {
-              priceCents: {
-                min: 22500,
-                max: 22500,
-              },
-            },
-            {
-              priceCents: {
-                min: 22500,
-                max: 22500,
-              },
-            },
-            {
-              priceCents: {
-                min: 29500,
-                max: 29500,
-              },
-            },
-            {
-              priceCents: {
-                min: 35000,
-                max: 35000,
-              },
-            },
-            {
-              priceCents: {
-                min: 36000,
-                max: 36000,
-              },
-            },
-          ],
-        },
+          artworks_connection: {
+            edges:
+              [
```

Comment:
Hmm shouldn't each element in the `edges` array be on object that has a `node` property?

I.e. I'd expect something like [this](https://user-images.githubusercontent.com/140521/66503303-f94ad080-ea94-11e9-87cb-311e819193cc.png)? But then, tests are working, so I'm not quite sure about this 🤔